.changeset/three-humans-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
```

Comment:
we don't need to do release for this since this is a fix in documentation and that gets published for all master merged. so can you remove this changeset?

Reply:
removed it